src/resmom_win/win/mom_mach.c

```diff
@@ -610,7 +610,7 @@ mem_sum(job *pjob)
 			ret = IsProcessInJob(ptask->ti_hProc, pjob->ji_hJob, &is_process_in_job);
 			if (!ret)
 				log_eventf(PBSEVENT_DEBUG3, PBS_EVENTCLASS_SERVER, LOG_DEBUG, __func__,
-					"IsProcessInJob failed for %d job %d process with errno %lu", pjob->ji_hJob, ptask->ti_hProc, GetLastError());
+					"IsProcessInJob failed for %s job %d process with errno %lu", pjob->ji_hJob, ptask->ti_hProc, GetLastError());
```

Comment:
why did you change here? Please check your format message because as per message i guess you are trying to print the jobid but passing the handle value to it.